pkg/tanka/evaluators.go

```diff
@@ -19,15 +19,16 @@ func EvalJsonnet(path string, opts jsonnet.Opts) (raw string, err error) {
 	// evaluate Jsonnet
 	if opts.EvalScript != "" {
 		evalScript := fmt.Sprintf(opts.EvalScript, entrypoint)
-		raw, err = jsonnet.Evaluate(entrypoint, evalScript, opts)
-		if err != nil {
-			return "", errors.Wrap(err, "evaluating jsonnet")
-		}
-	} else {
-		raw, err = jsonnet.EvaluateFile(entrypoint, opts)
+		raw, err = jsonnet.Evaluate("<tanka-evalScript>", evalScript, opts)
```

Comment:
It might be my lack of deep understanding about jsonnet internals, but for my own learning: How does this work? What is `<tanka-evalScript>` in this particular context?

Reply:
Sure!

In Jsonnet, there are two "types" of things that are evaluated: Anonymous snippets (this is the case here), and files imported by the importer. 

Anonymous snippets are supposed to have a filename, but it must not be resolvable by the importer. Problem in this case, our own `jsonnet.Evaluate` disregards this, so I had to take this change back for now. Will fix this in a future PR